test/MsBuildScript.Tests/runtests.target

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="RunMsBuildScriptTests" 
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="../../dir.props" />
```

Comment:
@nguerrera  @dsplaisted  I am experimenting "unit test" for msbuild script. So far it is pretty weak due to msbuild scope, do you like it or not? I can move this part